cron/k8s/transfer-v2.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Security Scorecard Authors
```

Comment:
Delete file?

Reply:
sorry I'm lost. Why do I need to delete this file? The `bq/main.go` is unchanged so I think we need this new instance. Please correct me if I'm wrong.

Reply:
Oh sorry. I confused this with the v2.release.yaml file. No need to delete this. Just update the job name to have `-v2` suffix.